plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -387,6 +387,19 @@ SecRule REQUEST_FILENAME "@endsWith /remote.php/dav/bulk" \
     ctl:ruleRemoveTargetById=944240;REQUEST_BODY,\
     ctl:ruleRemoveTargetById=944250;REQUEST_BODY"
 
+# Uploading files via Windows desktop client
+SecRule REQUEST_FILENAME "@rx /remote\.php/dav/uploads/[^/]+/[0-9]+/\.file$" \
```

Comment:
In the PR for the Android client you used `[a-z0-9]`, I believe. Is that necessary here too?

Reply:
a false positive wasn't reported within the a-z character range so it should be fine